src/collector.ts

```diff
@@ -132,6 +161,37 @@ class NaiveGomodParser {
             return dependencies;
         }, []);
 
+        function checkReplaceModule(dep: IDependency) {
```

Comment:
I think we can have simple function that can search and return replace module dependency if available, otherwise return null. Caller can handle the global list 'dependencies' not to distribute this global data among different function.